pilot/pkg/model/validation.go

```diff
@@ -1573,7 +1573,6 @@ func checkServiceRoleBinding(in *rbac.ServiceRoleBinding) error {
 		errs = appendErrors(errs, fmt.Errorf("exactly one of `roleRef`, `role`, or `actions` must be specified"))
 	}
 	if in.RoleRef != nil {
-		rbacLog.Warnf("`roleRef` is deprecated. Please use `role` instead")
```

Comment:
This also gets printed to stdout. I think we can remove this for now. `roleRef` is not the only deprecated thing, so later on we can add better logs for all deprecated features. @yangminzhu